pubtools/_pulp/task.py

```diff
@@ -89,14 +89,35 @@ def step(cls, name):
     def _basic_args(self):
         # minimum args required for a pulp CLI task
         self.parser.add_argument(
-            "--debug", action="store_true", help="show debug statements"
+            "--debug",
```

Comment:
"-d" argument could also be added here along with "--debug" so that multiple levels could be define with something like "-dd" or "-ddd" rather than writing "--debug" multiple times.

Reply:
Yep, makes sense => https://github.com/release-engineering/pubtools-pulp/pull/14.